aws/src/integration/java/org/apache/iceberg/aws/TestAssumeRoleAwsClientFactory.java

```diff
@@ -75,7 +75,7 @@ public void before() {
     assumeRoleProperties = Maps.newHashMap();
     assumeRoleProperties.put(AwsProperties.CLIENT_FACTORY, AssumeRoleAwsClientFactory.class.getName());
     assumeRoleProperties.put(AwsProperties.HTTP_CLIENT_TYPE, AwsProperties.HTTP_CLIENT_TYPE_APACHE);
-    assumeRoleProperties.put(AwsProperties.CLIENT_ASSUME_ROLE_REGION, "us-east-1");
+    assumeRoleProperties.put(AwsProperties.CLIENT_ASSUME_ROLE_REGION, AwsIntegTestUtil.testRegion());
```

Comment:
If the bucket is created in a different region, we need to use the region specified by the user to perform integration tests. Otherwise, we may get a 400 bad request error when sending a request from `us-east-1`